src/pybind/mgr/dashboard/frontend/src/app/shared/models/cd-table-selection.ts

```diff
@@ -22,7 +23,18 @@ export class CdTableSelection {
    * Get the first selected row.
    * @return {any | null}
    */
+
   first() {
     return this.hasSelection ? this.selected[0] : null;
   }
+
+  getIds() {
```

Comment:
This class is generic, OSD related code is prohibited. Please implement this using a different approach.